README.md

```diff
@@ -19,7 +19,35 @@ Remember: We _never_ create virtual environments in our _project_ directory!
 $ python3 -m venv <PATH_TO_DESIRED_LOCATION>
 ```
 
-2. Activate the virtual enironment
+2. Activate the virtual environment
 ```bash
 $ source <PATH_TO_VIRTUAL_ENV>/bin/activate
 ```
+
+
+### Creating a project
+
+1. Download Django by accessing the pip install from https://pypi.org/project/Django/
+
+2. Create a a new project. At this point you should be in your virtual environment.
+```bash
+$ django-admin startproject <NAME OF SITE>
+```
+
+3. Create a development server. Make sure you're in the sites directory.
+```bash
+$ python manage.py runserver
+```
+
+
+### Syncing database structure with the code from master
```

Comment:
The steps you've outlined here are for managing the code updates. The ticket asks for:

"The command(s) that I should run _after_ I pull the code changes from master in order for my database structure to be in sync with the code"

Check this page: https://docs.djangoproject.com/en/2.2/intro/tutorial02/#database-setup - skip down to the first example of a command that needs to be run in the terminal and read a little from there - it should ring a bell. Then you can also check here: https://docs.djangoproject.com/en/2.2/ref/django-admin/#django-admin-migrate (the django-admin command is the same as manage.py)